packages/pangea-sdk/PangeaCyber.Net/Redact/Requests/UnredactRequest.cs

```diff
@@ -0,0 +1,26 @@
+using Newtonsoft.Json;
+
+namespace PangeaCyber.Net.Redact
+{
+    /// <kind>class</kind>
+    /// <summary>
+    /// UnredactRequest
+    /// </summary>
+    public class UnredactRequest : BaseRequest
+    {
+        ///
+        [JsonProperty("redacted_data")]
+        public object RedactedData { get; set; }
```

Comment:
Would this not be the same `T` as the resulting `UnredactResult<T>`?

Reply:
Done